tests/Tests/Unit/ApiExceptionTest.php

```diff
@@ -243,4 +246,43 @@ public function getRpcStatusData()
             ]
         ];
     }
+
+    /**
+     * @dataProvider buildRequestExceptions
+     */
+    public function testCreateFromRequestException($re, $stream, $expectedCode) {
```

Comment:
cs-nit, but we should always use newlines for curly braces after function names:
```suggestion
    public function testCreateFromRequestException($re, $stream, $expectedCode)
    {
```

Reply:
Done!